ansible_base/activitystream/apps.py

```diff
@@ -0,0 +1,43 @@
+from functools import partial
+
+from django.apps import AppConfig
+from django.db.models.signals import m2m_changed, post_save, pre_delete, pre_save
+
+from ansible_base.activitystream.signals import activitystream_create, activitystream_delete, activitystream_m2m_changed, activitystream_update
+
+
+def connect_activitystream_signals(cls):
+    post_save.connect(activitystream_create, sender=cls, dispatch_uid=f'dab_activitystream_{cls.__name__}_create')
```

Comment:
Is there a reason we can't do pre_save for the create as well? I see we key off the created field but could we do something `if not pk` or is there a case like get_or_create where that would fall through?

Reply:
We need the PK of the newly created object when we make the activity stream entry. If we did this in pre_save, we'd have no way to get that.